templates/security.html.twig

```diff
@@ -16,7 +16,7 @@
 <div class="page page-center">
     <div class="container-tight py-4">
         <div class="text-center mb-4">
-            <h1>{% block logo_login %}Tabler{% endblock %}</h1>
+            {% block logo_login %}<h1>Tabler</h1>{% endblock %}
```

Comment:
Better make it a two block element:
```
{% block logo_login %}<h1>{% block title_login %}Tabler{% endblock %}</h1>{% endblock %}
```
I would even love to have it the other way around for BC reasons, but it think this block name order is more logical?

Reply:
I'm OK with the two block element.

```suggestion
            {% block logo_login %}<h1>{% block title_login %}Tabler{% endblock %}</h1>{% endblock %}
```

In my case logos will always be a SVG, `title_login` means title. Logic!